addons/ace_tweaks/ACE_Medical_Treatment.hpp

```diff
@@ -0,0 +1,13 @@
+
+class ace_medical_treatment {
+    class Medication {
+        class PainKillers {
```

Comment:
GVAR(PainKillers)

Reply:
This class is part of the unfinished ace implementation